js-core-1/week-3/lesson.md

```diff
@@ -88,11 +84,11 @@ Arrays also have several methods that you can use.
 _An array method that sorts the values in an array into ascending alphabetical or numerical order._
 
 ```js
-var unorderedLetters = ["z", "v", "b", "f", "g"];
-var orderedLetters = unorderedLetters.sort();
+let unorderedLetters = ["z", "v", "b", "f", "g"];
+let orderedLetters = unorderedLetters.sort();
 
-var unorderedNumbers = [8, 5, 1, 4, 2];
-var orderedNumbers = unorderedNumbers.sort();
+let unorderedNumbers = [8, 5, 1, 4, 2];
+let orderedNumbers = unorderedNumbers.sort();
```

Comment:
Is it worth mentioning with `sort` that it mutates the original array?

Reply:
That's a really good point - I think we should definitely mention or add a sentence or two around it

Reply:
@jamesgorrie , perhaps this can be exemplified with an exercise?